### DISCUSSION TITLE
Can we use Foocus generated images for commercial purposes like NFTs?

### DISCUSSION THREAD

**Discussion by thakurtech** [lllyasviel/Fooocus#3029] [Q&A] [answered]
Can we use Foocus generated images for commercial purposes like NFTs?

**Comment by ILikeToasters** [comment]
Really has nothing to do with Fooocus but the models used. SDXL itself is fine usually. All the newer models from Stability AI do require a membership. But I am not a lawyer and based upon my legal attempts to evict a tenant that has not paid rent in 6+ months myself in the court system I would probably not be very good at it.

**Comment by mashb1t** [comment, marked as answer]
see https://github.com/lllyasviel/Fooocus/discussions/1957
